externals/nghttp2/src/makeHook.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+make clean
+make
+make install
```

Comment:
Hm, I still think it's unnecessary.  Just after `make` there should already be a copy of libnghttp2.a somewhere in the tree.